### PR TITLE
fixed template vars, modified autoprefixer in helper

### PR DESCRIPTION
**Pull request by dominikwilkowski**
Quick fixes for module templates

### CODE REVIEW

File: .templates/new-module/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"name": "[-replace-name-]",
+	"name": "@gov.au/[-replace-name-]",
```
Comment (elisechant) [inline .templates/new-module/package.json]: `govau`? without `.`?

File: helper.js [hunk 2]
```diff
@@ -201,7 +201,7 @@ HELPER.compile = (() => {
 	const Autoprefix = ( file ) => {
 		const data = Fs.readFileSync( file, 'utf-8');
 
-		Postcss([ Autoprefixer ])
+		Postcss([ Autoprefixer({ browsers: ['last 2 versions', 'ie 8', 'ie 9', 'ie 10'] }) ])
```
Comment (elisechant) [inline helper.js]: you could use a browserslist file instead here

File: helper.js [hunk 3]
```diff
@@ -210,9 +210,9 @@ HELPER.compile = (() => {
 						console.warn( warn.toString() );
 				});
 
-			Fs.writeFileSync( file, prefixed.css );
+				Fs.writeFileSync( file, prefixed.css );
 
-			HELPER.log.success(`Autoprefixed file ${ Chalk.yellow( file ) }`);
+				HELPER.log.success(`Autoprefixed file ${ Chalk.yellow( file ) }`);
```
Comment (elisechant) [inline helper.js]: could be safer access to have `HELPER.log('success', 'message')` instead, making method a single level to HELPER

File: helper.js [hunk 4]
```diff
@@ -182,6 +181,7 @@ HELPER.compile = (() => {
 	const Sassify = ( scss, css ) => {
 		const compiled = Sass.renderSync({
 			file: scss,
+			indentType: 'tab',
```
Comment (elisechant) [inline helper.js]: SPACES! :trollface:

File: .templates/new-module/package.json (hunk 1 above)
Comment (dominikwilkowski) [inline .templates/new-module/package.json]: nope, We got'zz the dot'zz!
Proof here: https://twitter.com/npm_support/status/819316199083347968
and here: https://www.npmjs.com/org/gov.au

Comment (dominikwilkowski) [inline .templates/new-module/package.json]: nope, We got'zz the dot'zz!
Proof here: https://twitter.com/npm_support/status/819316199083347968
and here: https://www.npmjs.com/org/gov.au

File: helper.js (hunk 2 above)
Comment (dominikwilkowski) [inline helper.js]: good idea, might just do that :)

File: helper.js (hunk 3 above)
Comment (dominikwilkowski) [inline helper.js]: I find it more soothing to know what I'm passing here is going in the right debug hole :)

File: helper.js (hunk 4 above)
Comment (dominikwilkowski) [inline helper.js]: NEVER 🔥

File: helper.js (hunk 3 above)
Comment (elisechant) [inline helper.js]: if you're comfortable that your this code assumes `log` must exist

Comment (dominikwilkowski) [inline helper.js]: I am yeah. If it doesn't it should burn in a fire! :)
(This is _just_ the helper that is not distributed with the uikit)

File: helper.js (hunk 4 above)
Comment (elisechant) [inline helper.js]: Uh oh. SPACES! lol. 
Seriously though do you need this setting in combination with `outputStyle: 'compressed'`? will it even have an output?

File: helper.js (hunk 2 above)
Comment (elisechant) [inline helper.js]: you could reason either way. But I think if you wrap it in to a configurable constant or a browserslist file then users can be opinionated about which browsers their app cares about

File: helper.js (hunk 4 above)
Comment (dominikwilkowski) [inline helper.js]: No it won't but this bit of the helper will be replaced by pancake once it's done I assume so I just want them all to have the same settings for now.

Also: Tabs are literally the one and only semantic element for indentation. LITERALLY!

File: helper.js (hunk 2 above)
Comment (dominikwilkowski) [inline helper.js]: The user won't have control over this bit. You're confusing this with [pancake](https://github.com/govau/pancake). This helper only does some compiling for us now until pancake takes the cake... haha

Comment (dominikwilkowski) [inline helper.js]: I moved it there now: https://github.com/govau/pancake/issues/6

File: .templates/new-module/src/sass/globals.scss [hunk 5]
```diff
@@ -12,8 +12,6 @@
 //--------------------------------------------------------------------------------------------------------------------------------------------------------------
 // SASS VERSIONING
 //--------------------------------------------------------------------------------------------------------------------------------------------------------------
-@import '../../node_modules/sass-versioning/dist/index.scss';
```
Comment (klepas) [inline .templates/new-module/src/sass/globals.scss]: Think this got edited in one of my branches too — should be fine tho.

**Pull request merged**